package.json

```
@@ -44,53 +43,22 @@
         "typescript": "5.1.6",
         "yup": "^1.4.0"
     },
-    "overrides": {
-        "react-scripts": {
-            "typescript": "$typescript"
-        }
-    },
     "scripts": {
-        "start": "PORT=3002 react-scripts start",
-        "build": "react-scripts build",
-        "test": "react-scripts test --watchAll=false",
-        "test:watch": "react-scripts test",
-        "test:coverage": "react-scripts test --coverage",
-        "eject": "react-scripts eject",
+        "start": "vite",
+        "start:open": "vite --open",
+        "build": "tsc && vite build",
+        "serve": "vite preview",
+        "test": "jest",
         "lint": "eslint . --ext js,mjs,jsx,ts,mts,tsx --max-warnings 0"
     },
-    "jest": {
-        "moduleNameMapper": {
-            "\\.svg": "<rootDir>/src/__mocks__/svgrMock.js"
-        },
-        "transformIgnorePatterns": [
-            "node_modules/(?!@gridsuite/commons-ui)/"
-        ]
-    },
-    "eslintConfig": {
-        "extends": [
-            "react-app",
-            "plugin:prettier/recommended"
-        ],
-        "rules": {
-            "prettier/prettier": "warn",
-            "curly": "error"
-        }
-    },
-    "browserslist": {
-        "production": [
-            ">0.2%",
-            "not dead",
-            "not op_mini all"
-        ],
-        "development": [
-            "last 1 chrome version",
-            "last 1 firefox version",
-            "last 1 safari version"
-        ]
-    },
     "devDependencies": {
+        "@babel/helper-builder-react-jsx": "7.22.10",
         "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
+        "@babel/preset-env": "^7.24.3",
+        "@babel/preset-react": "^7.24.1",
+        "@babel/preset-typescript": "^7.24.1",
         "@types/core-js": "^2.5.8",
+        "@types/eslint": "^7.29.0",
```

`@types/*` must have major part of version in sync with the lib
```suggestion
        "@types/eslint": "^8.56.9",
```

---

Indeed, so you forgot to it here right ? :p
https://github.com/gridsuite/gridstudy-app/pull/2045/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R82